### PR TITLE
add appconfiguration tests

### PR DESCRIPTION
**Pull request by Youri970410**

### CODE REVIEW

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/AppConfigurationClientBase.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        public ResourceGroupsOperations ResourceGroupsOperations { get; set; }
+        public PrivateLinkResourcesOperations PrivateLinkResourcesOperations { get; set; }
+        public Operations Operations { get; set; }
+        public string AZURE_LOCATION { get; set; }
+        public string KEY_UUID { get; set; }
+        public string LABEL_UUID { get; set; }
+        public string KEY { get; set; }
+        public string LABEL { get; set; }
+        public string TEST_CONTENT_TYPE { get; set; }
+        public string TEST_VALUE { get; set; }
```
Comment (erich-wang) [inline sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/AppConfigurationClientBase.cs]: please change names to follow C# name guideline, e.g. TestValue

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("Microsoft.appconfiguration")]
```
Comment (erich-wang) [inline sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Properties/AssemblyInfo.cs]: please use "Microsoft.AppConfiguration"

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/AppConfigurationTest.cs [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        }
+
+        [Test]
+        public async Task Test_AppConfiguration()
+        {
+            //string SERVICE_NAME = "myapimrndxyz";
+            //string VNET_NAME = "vnetname";
+            //string SUB_NET = "subnetname";
+            //string ENDPOINT_NAME = "endpointxyz";D:\sdk\20200507\azure-sdk-for-net\sdk\appconfiguration\Azure.Management.AppConfiguration\src\Generated\Operations\PrivateEndpointConnectionsRestClient.cs
+            string configuration_Store_Name = Recording.GenerateAssetName("configuration");
```
Comment (erich-wang) [inline sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/AppConfigurationTest.cs]: Please don't use understore in the name

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/AppConfigurationTest.cs [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            //create Key
+            //var addConfigurationsetting = await PrivateEndpointConnectionsClient
+            //[skip]
+            //var listkeyvalueResult = await ConfigurationStoresClient.ListKeyValueAsync(resourceGroup, CONFIGURATION_STORE_NAME,KEY,LABEL);
+            //# ConfigurationStores_ListKeyValue[post]
+            //    BODY = {
+            //        "key": KEY,
+            //  "label": LABEL
+            //}
+            //    result = self.mgmt_client.configuration_stores.list_key_value(resource_group.name, CONFIGURATION_STORE_NAME, BODY)
```
Comment (erich-wang) [inline sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/AppConfigurationTest.cs]: Should we keep these commented lines? Let's talk in the meeting

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (Youri970410) [reply sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Properties/AssemblyInfo.cs]: done

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/AppConfigurationTest.cs (hunk 3 above)
Comment (Youri970410) [reply sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/AppConfigurationTest.cs]: got it

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/AppConfigurationTest.cs (hunk 4 above)
Comment (Youri970410) [reply sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/AppConfigurationTest.cs]: ok,thanks

**Pull request merged**